docs/rest_api2.rst

```diff
@@ -0,0 +1,4259 @@
+|                    | {                                                                                                                                                                           |
+|                    |                                                                                                                                                                             |
+|                    | "interfaces": [                                                                                                                                                             |
+|                    |                                                                                                                                                                             |
+|                    | {                                                                                                                                                                           |
+|                    |                                                                                                                                                                             |
+|                    | "path": "Pod-1/Node-101/eth1/1",                                                                                                                                            |
+|                    |                                                                                                                                                                             |
+|                    | "name": "internal"                                                                                                                                                          |
+|                    |                                                                                                                                                                             |
+|                    | },                                                                                                                                                                          |
+|                    |                                                                                                                                                                             |
+|                    | {                                                                                                                                                                           |
+|                    |                                                                                                                                                                             |
+|                    | "path": "Pod-1/Node-101/eth1/2",                                                                                                                        
```

Comment:
..VLAN information for specific VLANs <or 'a specific VLAN'> from a BIG-IP Device to the F5 ACI...